### ISSUE TITLE
[1.12.2] recent beta update version does not seem to save version number properly in `dsurround.cfg`

### ISSUE THREAD

**Issue by ubergarm** [OreCruncher/DynamicSurroundings#775]
## Description
Not a big deal, but I just noticed after updating from the previous version in my modpack [Rotten Economy](https://www.curseforge.com/minecraft/modpacks/rotten-economy/) that the new `dsurround.cfg` is generating as `@VERSIONS@` instead of `3.6.3` like I would expect.

It does not effect me at all afaict, but perhaps it matters for something for you?

snippet from the `git diff` after updating:
```
--- a/config/dsurround/dsurround.cfg
+++ b/config/dsurround/dsurround.cfg
@@ -1,6 +1,6 @@
 # Configuration file

-~CONFIG_VERSION: 3.6.2.1
+~CONFIG_VERSION: @VERSION@
```

Cheers!

## Versions
| md5sum | filename version | name |
| --- | --- | --- |
| `02fcc447bb8938e5214292e4d36ec949` | `forge-1.12.2-14.23.5.2855.jar` | "forge" |
| `77e815ce9c8dd68ecba4eb884d6db4d2` | `DynamicSurroundings-1.12.2-3.6.3.jar` | "new" / "latest" |
| `73cd6e10ae80337b8e1680806a34cb93` |  `DynamicSurroundings-1.12.2-3.6.2.1.jar` | "old" / "previous" |

**Comment by OreCruncher** [comment]
Yep - know issue.  Going to be fixed the next drop.  Also the fingerprint is going away.

**Comment by janagyjr** [comment]
Is there a workaround for this until an update is pushed? I'm still having this issue.
